fabfile.py

```diff
@@ -150,13 +148,13 @@ def update_refinery():
 def relaunch_refinery(dependencies=False, migrations=False):
     """Perform a relaunch of a Refinery Platform instance, including processing
     of grunt tasks
-    dependencies: update bower and pip dependencies
+    dependencies: update npm and pip dependencies
     migrations: apply migrations
     """
     puts("Relaunching Refinery")
     with cd(os.path.join(env.refinery_app_dir, "ui")):
         if dependencies:
-            run("bower update --config.interactive=false")
+            run("npm update --config.interactive=false")
```

Comment:
This should probably be `npm update --progress false`. However, I would just remove `relaunch_refinery()` - I think it has not been used in a long time.

Reply:
I've never used it, so I can remove it.